### DISCUSSION TITLE
New to kafka and learning. Use of examples...

### DISCUSSION THREAD

**Discussion by QuidneIT** [castorm/kafka-connect-http#110] [Q&A] [answered]
Good afternoon. 

The 2 search examples you posted with this, how do you use them as I believe you can not put that text in the properties file?

Raoul.

**Comment by QuidneIT** [comment, marked as answer]
Problem resolved. REST API documentation was wrong in telling me to format the body as text-plain, while it needed json format.

**Reply by castorm** [reply]
Hi @QuidneIT,

I'm afraid the examples you followed were meant to be used differently. As you said Kafka Connect's REST API is JSON based.

I'm happy you solved this.
